sensorserviceutils/sensorserviceutils.go

```diff
@@ -313,7 +307,8 @@ func Parse(f string, meta *bugreportutils.MetaInfo) OutputData {
 			continue
 		}
 	}
-	p.creatUnseenActiveConnectionHistory()
+	p.createUnseenActiveConnectionHistory()
```

Comment:
What is the difference between createUnseenActiveConnectionHistory & createHistoryForEventsWithNoActivation

Reply:
For createUnseenActiveConnectionHistory, it deals with the active connections that have no activation statement seen in the history.
For createHistoryForEventsWithNoActivation, it deals with the case where we only see de-activation statement of an event, but no activation statement due to the limited history.